tailcallhq/tailcall
# fix(2775): send metrics to posthog

fixes: #2775
/claim #2775

<img width="1733" alt="Screenshot 2024-09-01 at 5 44 44 PM" src="https://github.com/user-attachments/assets/30e7d9ed-699b-49f6-ba2c-c73770fcbf59">

---
Steps to test:

1. Add posthog project api key to env `POSTHOG_API_SECRET`.
2. For local testing I had to hardcode `is_tracking` to true.
3. Run cargo test on tailcall-tracker or just use build the project and run any argument.
4. Check the activity section in posthog project events should appear.
